examples/evm-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -27,15 +27,15 @@ const getStatus = async (
 };
 
 export async function erc20Transfer(): Promise<void> {
-  const provider = new providers.JsonRpcProvider(CRONOS_RPC_URL);
+  const provider = new providers.JsonRpcProvider(DESTINATION_RPC_URL);
```

Comment:
it transfer is from `source` network, why we rename then env var as `SOURCE...`?